### PR TITLE
changed wcs order to remain as astropy wcs and added the new cubeseq.

### PR DESCRIPTION
**Pull request by abit2**
changed the 6 letters of wcs class :P and added the cubesequence code

### CODE REVIEW

File: sunpycube/cube/datacube.py [hunk 1]
```diff
@@ -495,11 +495,29 @@ def _array_is_aligned(self):
...
+    """
+    Class representing list of cubes.
+
+    Attributes
+    ----------
+    data_list : `list`
+        List of cubes.
+    """
+
+    def __init__(self, data_list):
```
Comment (Cadair) [inline sunpycube/cube/datacube.py]: This would be better being `(self, *data_list)`

File: sunpycube/cube/datacube.py [hunk 2]
```diff
@@ -495,11 +496,37 @@ def _array_is_aligned(self):
...
+class CubeSequence(object):
+    """
+    Class representing list of cubes.
+
+    Attributes
+    ----------
+    data_list : `list`
+        List of cubes.
+
+    meta : `astropy.io.fits.header.Header` or None
```
Comment (DanRyanIrish) [inline sunpycube/cube/datacube.py]: For the time being make ```meta: `dict` or None```

File: sunpycube/cube/datacube.py [hunk 3]
```diff
@@ -495,11 +496,37 @@ def _array_is_aligned(self):
...
+
+    def __init__(self, data_list, meta=None):
+        if not all(isinstance(data, Cube) for data in data_list):
+            raise ValueError("data list should be of cube object")
+
+        if not isinstance(meta, astropy.io.fits.header.Header):
+            raise ValueError("meta shoulld be astropy.io.fits.header.Header instance")
+
+        self.data = data_list
+        if meta is not None:
```
Comment (DanRyanIrish) [inline sunpycube/cube/datacube.py]: Again, for now just replace this if statement with ```self.meta = meta```

File: sunpycube/cube/cube_utils.py [hunk 4]
```diff
@@ -50,8 +50,8 @@ def orient(array, wcs, *extra_arrs):
     else:
         array_order = select_order(axtypes)
     result_array = array.transpose(array_order)
-    wcs_order = np.array(select_order(axtypes))[::-1]
-
+    wcs_order = np.array(select_order(axtypes))
+
```
Comment (DanRyanIrish) [inline sunpycube/cube/cube_utils.py]: PEPE8: Delete extra spaces on this blank line.

File: sunpycube/cube/cube_utils.py [hunk 5]
```diff
@@ -503,6 +503,20 @@ def _convert_slice(item, wcs, axis, _source='cube'):
...
+    Parameters
+    ----------
+    cubesequence: sunpycube.CubeSequence object
+        The cubesequence to get the item from
+    item: int, slice object, or tuple of these
+        The item to get from the cube
+    """
+    if isinstance(item, int):
+        return cubesequence.data[item]
+    return cubesequence.data[item[0]][item[1::]]
```
Comment (DanRyanIrish) [inline sunpycube/cube/cube_utils.py:525]: From our previous conversation on riot, we want the final index to be the cube index.  So don't we want this line to be ```return cubesequence.data[item[-1]][item[:-1:]]``` ?

Comment (DanRyanIrish) [reply sunpycube/cube/cube_utils.py:525]: Forget this comment.

File: sunpycube/cube/datacube.py [hunk 6]
```diff
@@ -16,6 +16,7 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import astropy.nddata
+import astropy.io.fits.header
```
Comment (DanRyanIrish) [inline sunpycube/cube/datacube.py]: This import is not used, right?  If not, delete.

File: sunpycube/cube/datacube.py [hunk 7]
```diff
@@ -495,11 +496,32 @@ def _array_is_aligned(self):
     def __getitem__(self, item):
         if item is None or (isinstance(item, tuple) and None in item):
             raise IndexError("None indices not supported")
-        if not self._array_is_aligned():
-            raise cu.CubeError(6, "Slicing on unaligned wcs-array systems " +
-                               "not supported at the moment")
```
Comment (DanRyanIrish) [inline sunpycube/cube/datacube.py:500]: Can you remind me why we deleted this if statement?

Comment (abit2) [reply sunpycube/cube/datacube.py:500]: it was failing here and you said something about physical coordinates @DanRyanIrish

Comment (DanRyanIrish) [reply sunpycube/cube/datacube.py:500]: That's right!  Thanks!

**Comment by DanRyanIrish** [comment]
👍 I'm happy to merge this as is and then do future work in another PR.  Any other reviews/thumbs up?  @Cadair @ehsteve ?

**Review by Cadair — APPROVED** [review]

**Comment by Cadair** [comment]
This is going to break *all the shit* once you write tests.

**Pull request merged**